apps/snitch_core/test/tools/mailer/order_confirmation_email_test.exs

```diff
@@ -0,0 +1,35 @@
+defmodule Snitch.Tools.OrderEmailTest do
+  use ExUnit.Case, async: true
+  use Snitch.DataCase
+  use Bamboo.Test
+
+  import Snitch.Factory
+  alias Snitch.Tools.OrderEmail
+
+  setup :order_with_user
+  setup :order_with_lineitem
+
+  test "send order confirmation mail succesfully", context do
+    config = insert(:general_config)
+    %{order_with_lineitem: line_item} = context
```

Comment:
Can we do this match in line 12 only, like `test "send order confirmation mail succesfully", %{order_with_lineitem: line_item} = context do`? That would save you one extra line of code ? what do you think? 

This applies to all the test cases in this file.